appium-dotnet-driver/Appium/AppiumDriver.cs

```diff
@@ -431,6 +429,38 @@ public object GetSessionDetail(string detail)
 
         #endregion Session Data
 
+        #region Device Methods
+
+        /// <summary>
+        /// Start recording the device screen
+        /// </summary>
+        /// <param name="opts">recording options like the following:
+        ///  Dictionary<string, object> StartRecordOptions = new Dictionary<string, object>();
+        ///  StartRecordOptions.Add("bit_rate", "1");

```

Comment:
Please add a proper options documentation:

https://github.com/appium/appium-android-driver/blob/master/lib/commands/recordscreen.js
https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/recordscreen.js

Reply:
Sure thing, will add in.

Reply:
@mykola-mokhnach  just noticed the docs on http://appium.io/docs/en/commands/device/recording-screen/start-recording-screen/ don't match what you linked. The JSON parameters are underscore vs camelCase. Should i go by camelcase since that's what it was implemented in?

Reply:
camelCase is correct. Let me update the doc. Thanks!
e.g. 
https://github.com/appium/ruby_lib_core/blob/master/test/unit/android/device/w3c/commands_test.rb#L229